firmware/gameport-adapter/gameport-adapter.ino

```diff
@@ -54,8 +61,10 @@ static Joystick *createJoystick() {
 }
 
 void setup() {
-  Serial.begin(9600);
-  //while(!Serial);
+  #ifndef NDEBUG
+    Serial.begin(9600);
+    while(!Serial); 
```

Comment:
I just got an idea to move this code into Utilities as well, let's call it InitLogging() and in the Utilities this function would again either do something or not depending on the NDEBUG macro. This way, we would have the control of that stuff all in one place.

Reply:
Good idea, added a InitLogging() function during the default setup() routine and implemented it in Utilities.h